src/gui/kb.cpp

```diff
@@ -283,6 +303,7 @@ void Kb::load(){
                 newCurrentProfile = profile;
         }
     }
+    showBatteryIndicator = QString::compare(settings.value("batteryIndicator").toString().trimmed().toLower(), "yes") == 0;
```

Comment:
This can be simplified if you just use a bool.
showBatteryIndicator = settings.value("batteryIndicator", true).toBool();

Reply:
Interestingly enough, this is more equivalent to `!(QString::compare(settings.value("batteryIndicator").toString().trimmed().toLower(), "false") == 0);`

Because of that, keyboards would load with an enabled tray icon, which would lead to a segfault.
That would've happened before if I edited my conf file to enable the icon on a keyboard, so that'll be fixed too :sweat_smile: